src/js/views/patients/sidebar/action/action-sidebar_views.js

```diff
@@ -128,6 +134,14 @@ const LayoutView = View.extend({
   initialize({ action }) {
     this.action = action;
   },
+  onAttach() {
+    anime({
+      targets: this.el,
+      translateX: [{ value: 20, duration: 0 }, { value: 0, duration: 200 }],
+      opacity: [{ value: 0, duration: 0 }, { value: 1, duration: 300 }],
+      easing: 'easeInOutQuad()',
```

Comment:
in other places we're not using the `()` so I think it's safe to remove it.. I _think_ that might be deprecated if I'm remembering correctly